docs/judge/problem_format.md

```diff
@@ -1,19 +1,19 @@
-Each problem is stored in its own directory. That directory must contain a file named `init.json`.
+Each problem is stored in its own directory. That directory must contain a file named `init.yml`.
 
-# `init.json`
-The entire file is a JSON Object. It must contain one key, `test_cases`, which maps to a list of test cases. Optionally, but almost always, will there be an `archive` key, which allows the problem data to be stored, compressed, in a `.zip` file, instead of the problem directory as flat files.
+# `init.yml`
+The entire file is a YAML object. It must contain one key, `test_cases`, which maps to a list of test cases. Optionally, but almost always, will there be an `archive` key, which allows the problem data to be stored, compressed, in a `.zip` file, instead of the problem directory as flat files.
```

Comment:
In YAML with `a: b`, I'm pretty sure that `a` is the key, `:` is the value indicator, and `b` is the value